CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ### Fixed
 
 - We fixed wrong button order (Apply and Cancel) in ManageProtectedTermsDialog.
+- We fixed an issue with incompatible characters at BibTeX-Key [#6257](https://github.com/JabRef/jabref/issues/6257)
+- We fixed an issue with incompatible characters at BibTeX key [#6257](https://github.com/JabRef/jabref/issues/6257)
+- We fixed an issue where dash (`-`) was reported as illegal BibTeX key [#6295](https://github.com/JabRef/jabref/issues/6295)
```

Comment:
Please add the removal of the "enforce legal key" option under "Removed".

Reply:
Should i also add as 'Added' that unwanted characters can be user-specified in preferences?

Reply:
yes, please!